tests/scripts/thread-cert/Cert_5_6_03_NetworkDataRegisterAfterAttachLeader.py

```diff
@@ -109,6 +116,44 @@ def test(self):
             if addr[0:10] == '2001:2:0:1' or addr[0:10] == '2001:2:0:2':
                 self.assertTrue(self.nodes[LEADER].ping(addr))
 
+    def verify(self, pv):
+        pkts = pv.pkts
+        pv.summary.show()
+
+        ROUTER = pv.vars['ROUTER']
+        MED = pv.vars['MED']
+        SED = pv.vars['SED']
+        _rpkts = pkts.filter_wpan_src64(ROUTER)
+
+        _rpkts_med = _rpkts.copy()
+        _rpkts_sed = _rpkts.copy()
+        # Step 3: The DUT MUST multicast a MLE Data Response for each
```

Comment:
Are we checking that the MLE Data Response is sent via multicast?

Reply:
added multicast dst